src/framework/auth/providers/dummy-auth.provider.ts

```diff
@@ -63,7 +70,11 @@ export class NbDummyAuthProvider extends NbAbstractAuthProvider {
         ['Something went wrong.']);
     }
 
-    // TODO is it missed messages here, is it token should be defined
-    return new NbAuthResult(true, this.createSuccessResponse(data), '/', ['Successfully logged in.']);
+    return new NbAuthResult(true,
```

Comment:
Why don't we just pass an object in this constructor instead of the multiple params?
I think it would be much prettier.

Reply:
Check todo